### PR TITLE
Update distribution: Java, Play! and Nginx

### PR DESCRIPTION
**Pull request by gregorkistler**
Made some smaller adjustments to fix boot up errors or warnings in Play! and Nginx

### CODE REVIEW

**Comment by xabolcs** [comment]
👌

File: docker/run.sh [hunk 1]
```diff
@@ -42,4 +42,4 @@ if [ -z "$SYSFS_CGROUP_FILES" ]; then
 	fi;
 fi;
 
-exec /play/play run -XshowSettings:vm -XX:+UseSerialGC -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
+exec /play/play run -XshowSettings:vm -XX:+UseSerialGC -XX:+UnlockExperimentalVMOptions
```
Comment (xabolcs) [inline docker/run.sh]: See my comment in the original issue!

Just keep `-XshowSettings:vm`, but please verify that the Docker container still accounts cgroup limits when you bind mount them.
See commit 9fdf38ed8db95527eb43c3a31a0684be14472643.

Comment (xabolcs) [reply docker/run.sh]: While at it, please rephrase the commit message too! Thanks

E.g. "Docker: fix broken and xxxxx (legacy?) JVM options for Java 11" and fill the details, like I did in commit ff77348a2adaf1eea63bccb292240592ce49e8ea!

**Comment by xabolcs** [comment]
supervisord spits out some `CRITICAL` log about config hardening ... I'll look into it, if you wouldn't be quicker! 

```
/usr/lib/python2.7/dist-packages/supervisor/options.py:461: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
  'Supervisord is running as root and it is searching '
2022-05-02 06:42:21,234 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-05-02 06:42:21,234 INFO Included extra file "/etc/supervisor/conf.d/supervisord.conf" during parsing
2022-05-02 06:42:21,243 INFO RPC interface 'supervisor' initialized
2022-05-02 06:42:21,243 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2022-05-02 06:42:21,243 INFO supervisord started with pid 7
```

File: docker/run.sh (hunk 1 above)
Comment (gregorkistler) [reply docker/run.sh]: Couldn't amend the last commit as I did it via Github Web and there is no such option yet 🤔

**Comment by gregorkistler** [comment]
Feel free to fix the supervisord issue :)

File: docker/run.sh (hunk 1 above)
Comment (xabolcs) [reply docker/run.sh]: OK, I'll squash it for you!

I'll rebase all the PR on my branch anyway! :)

Comment (gregorkistler) [reply docker/run.sh]: Thanks :)

**Comment by xabolcs** [comment]
Thank you, for the fixes!

Landed as ...
- 1c2460aa2a4b562b5d4b9acffe73b3813be35d90,
- 5badbbab3f0e3bf25dc2ad632644b9b26e1fa10f and
- 160f80f7d9c30adf155742d48cb6997f2e737ab4

**Pull request closed without merging**